junit-platform-console/src/main/java/org/junit/platform/console/options/TestDiscoveryOptionsMixin.java

```diff
@@ -39,7 +40,9 @@ class TestDiscoveryOptionsMixin {
 	@ArgGroup(validate = false, order = 2, heading = "%n@|bold SELECTORS|@%n%n")
 	SelectorOptions selectorOptions;
 
-	@ArgGroup(validate = false, order = 3, heading = "%n@|bold FILTERS|@%n%n")
+	@ArgGroup(validate = false, order = 3, heading = "%n  For more information on selectors including syntax examples, see"
+			+ "%n  @|underline https://junit.org/junit5/docs/current/user-guide/#running-tests-discovery-selectors|@"
```

Comment:
```suggestion
			+ "%n  @|underline https://junit.org/junit5/docs/${junit.docs.version}/user-guide/#running-tests-discovery-selectors|@"
```

I tried using a dynamic version here but hit https://github.com/remkop/picocli/issues/2281